cmd/tempo-vulture/main.go

```diff
@@ -45,95 +42,148 @@ func init() {
 	flag.StringVar(&prometheusPath, "prometheus-path", "/metrics", "The path to publish Prometheus metrics to.")
 	flag.StringVar(&prometheusListenAddress, "prometheus-listen-address", ":80", "The address to listen on for Prometheus scrapes.")
 
-	flag.StringVar(&lokiBaseURL, "loki-base-url", "", "The base URL (scheme://hostname) at which to find loki.")
-	flag.StringVar(&lokiQuery, "loki-query", "", "The query to use to find traceIDs in Loki.")
-	flag.StringVar(&lokiUser, "loki-user", "", "The user to use for Loki basic auth.")
-	flag.StringVar(&lokiPass, "loki-pass", "", "The password to use for Loki basic auth.")
-
-	flag.StringVar(&tempoBaseURL, "tempo-base-url", "", "The base URL (scheme://hostname) at which to find tempo.")
+	flag.StringVar(&tempoQueryURL, "tempo-query-url", "", "The URL (scheme://hostname) at which to query Tempo.")
+	flag.StringVar(&tempoPushURL, "tempo-push-url", "", "The URL (scheme://hostname) at which to push traces to Tempo.")
 	flag.StringVar(&tempoOrgID, "tempo-org-id", "", "The orgID to query in Tempo")
-	flag.DurationVar(&tempoBackoffDuration, "tempo-backoff-duration", time.Second, "The amount of time to pause between tempo calls")
+	flag.DurationVar(&tempoBackoffDuration, "tempo-backoff-duration", time.Second, "The amount of time to pause between Tempo calls")
 }
 
 func main() {
 	flag.Parse()
 
-	glog.Error("Application Starting")
+	glog.Error("Tempo Vulture Starting")
 
-	testDurations := []time.Duration{
-		24 * time.Hour,
-		12 * time.Hour,
-		6 * time.Hour,
-		3 * time.Hour,
-		time.Hour,
-		30 * time.Minute,
-	}
+	startTime := time.Now().UTC().UnixNano()
+	ticker := time.NewTicker(tempoBackoffDuration)
+	iteration := int64(0)
 
-	ticker := time.NewTicker(15 * time.Second)
+	// Write
 	go func() {
 		for {
 			<-ticker.C
 
-			for _, duration := range testDurations {
+			iteration++
+			rand.Seed(startTime / iteration)
```

Comment:
i was thinking of seeding with something like:
```
time.Now().Unix() / 5 * 5
```
this will floor the unix timestamp to the nearest 5 seconds.  If we start a goroutine every 5 seconds with the seed passed in we can generate a trace using the seed.  5 seconds could be configurable.